### PR TITLE
refactor: added favicon and og property

### LINKED ISSUE #44: Add a icon and og property
- Add of favicon to website.
    image

-   Add four basic open graph properties / meta data

### PR DESCRIPTION
**Pull request by rahmlad-aramide**
I've added an Icon and linked it to the favicon, it's now displaying as the below when opened in browser
after-browser

Also, I've added the og properties, attached is the image of the codebase of the before my update and after I've made the update:

Before
after-code

Resolves #44

### CODE REVIEW

**Review by malivinayak — APPROVED** [review]
LGTM

**Comment by malivinayak** [comment]
Because of updated feature this branch is give conflicts.
Because of the conflicts I am unable to merge this branch
Rebase the branch and resolve conflicts so that I can merge this PR

**Review by malivinayak — CHANGES_REQUESTED** [review]
OG properties are missing in index.html

**Comment by rahmlad-aramide** [comment]
I've also fixed the merge conflict, it has no conflict with the base branch anymore, kindly review

File: index.html [hunk 1]
```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 
+
```
Comment (malivinayak) [inline index.html:4]: remove extra blank line

File: index.html [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 <head>
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta property="og:title" content="To-Do-Manager">
+  <meta property="og:description" content="To-Do-Manager is a web app used to maintain our day-to-day tasks or list everything that we have to do. This project provides an easy way to add, edit and delete tasks. Added tasks will be stored in the browser’s local storage so, they won’t be removed on page refresh or tab close.">
+  <meta property="og:image" content="./assets/images/to-do icon.png">
+  <meta property="og:url" content="https://malivinayak.github.io/To-Do-Manager">
   <title>To-Do-Manager</title>
```
Comment (malivinayak) [inline index.html:13]: icon is not added
previously you have added icon but after your latest commit icon is not present
kindly verify it and make necessary changes

**Review by malivinayak — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -1,13 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    
-    <title>To-Do-Manager</title>
-    <!-- Favicon icon link -->
-    <link rel="icon" href="./assets/images/to-do icon.png" type="image/x-icon">
```
Comment (malivinayak) [inline index.html]: merge commit [b10c644](https://github.com/malivinayak/To-Do-Manager/pull/50/commits/b10c644729fef9f3a05b305624be98de87018720) removed the icon code

**Review by malivinayak — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (rahmlad-aramide) [reply index.html:13]: Oh, is that so? Alright, let me do a final check and make the fixes as requested.
@malivinayak

**Comment by rahmlad-aramide** [comment]
@malivinayak kindly review and merge as appropriate, all issues resolved

**Review by malivinayak — APPROVED** [review]
LGTM!

**Pull request merged**